src/WebAsmReader.h

```diff
@@ -0,0 +1,124 @@
+#ifndef WEBASMREADER_H
+#define WEBASMREADER_H
+
+
+#include <sys/stat.h>
+#include "support/file.h"
+#include "wasm-binary.h"
+#include "wasm-s-parser.h"
+#include "wasm-printing.h"
+
+using namespace cashew;
+using namespace wasm;
+
+struct WebAsmReader
```

Comment:
please follow the style conventions in other files. here, the `{` should be on the same line as the `struct`.